src/styles/utils/calendar-modification.scss

```diff
@@ -0,0 +1,27 @@
+@for $day from 1 through 31 {
+  .calendar__day:nth-child(#{$day})::before {
+    content: "#{$day}";
+  }
+}
+
+@each $day in $daysStart {
+  .calendar--start-day--#{$day} {
+    .calendar__day:first-child {
+      margin-left: if($day == mon, 0,
+      if($day == tue, $cell-size + 1,
+      if($day == wed, ($cell-size * 2) + 2,
+      if($day == thu, ($cell-size * 3) + 3,
+      if($day == fri, ($cell-size * 4) + 4,
+      if($day == sat, ($cell-size * 5) + 5,
+      $cell-size * 6 + 6))))));
```

Comment:
It's still not fixed 🤷 
If you have questions or don't know how to do it don't hesitate to ask in Q&A or in chat
For example, you can do this, modify a dayStart variable so you understand the position in the calendar and set a margin depending on it:
```scss
// Modify daysStart variable to store info about the position
$daysStart: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5 , sun 6;

@each $day , $num in $daysStart {
  // Use scss nesting and you can easily get a first-child selector
  &--start-#{$day} :first-child {
    margin-left: ($cell-size * $num) + $num;
  }
}
```

Reply:
> Yes, I had questions, and I went to a Q&A Zoom meeting. In the future, I will ask in the chat. Thanks for the answer.

Great! if Q&A is on you can ask the mentor directly. If it's not soon, don't wait, ask in the chat